README.rdoc

```diff
@@ -2,9 +2,14 @@
 
 ==Basics
 
-Wagn is an open web platform for building coherent web systems.
+Wagn is serious web development made fun.
```

Comment:
Yeah, that came up in working on some Decko marketing ideas, but no reason not to introduce that now.